### ISSUE TITLE
First horizontal line not rendering when UI at 90%

### ISSUE THREAD

**Issue by humberto-garza** [BoostIO/Boostnote#3211]
# Current behavior

On any page, when you add a horizontal line, either using ` *** ` or ` <hr> `  it does not show on the rendered view if it is the first one. To work around this, you need to add a second `<hr>`. Interestingly, when you copy-paste the rendered version of the md somewhere else, you see that the 2 lines are there.

# Expected behavior
![image](https://user-images.githubusercontent.com/8660999/62879907-c03eeb80-bcf1-11e9-95cd-72ea0743d858.png)

# Steps to reproduce

```md
# Test

***

Text a b c

***
```

![image](https://user-images.githubusercontent.com/8660999/62879855-a56c7700-bcf1-11e9-8fc4-c047faa575f3.png)

# Environment

- Version : 0.12.1
- OS Version and name : W10x64

This only happens when the zoom option is set to 90%; when it is set to 80%, 100%, 110%... the problem goes away... So I think it is a zoom related problem

# Interface settings
![image](https://user-images.githubusercontent.com/8660999/62885989-c9cf5000-bcff-11e9-9dbc-607616181143.png)

**Comment by Flexo013** [comment]
I can't seem to be able to reproduce this:
![image](https://user-images.githubusercontent.com/4507015/62882860-197b3f00-bd33-11e9-8e48-452239d77f93.png)

This is with 0.12.1 on Windows 10.

@humberto-garza How does this note look on your side?
```md
---
title: Actual frontmatter title
---

# Title

***

Text a b c

***

```

**Comment by humberto-garza** [comment]
> I can't seem to be able to reproduce this:
> ![image](https://user-images.githubusercontent.com/4507015/62882860-197b3f00-bd33-11e9-8e48-452239d77f93.png)
> 
> This is with 0.12.1 on Windows 10.
> 
> @humberto-garza How does this note look on your side?
> 
> ```lisp
> ---
> title: Actual frontmatter title
> ---
> 
> # Title
> 
> ***
> 
> Text a b c
> 
> ***
> ```

**Comment by humberto-garza** [comment]
> I can't seem to be able to reproduce this:
> ![image](https://user-images.githubusercontent.com/4507015/62882860-197b3f00-bd33-11e9-8e48-452239d77f93.png)
> 
> This is with 0.12.1 on Windows 10.
> 
> @humberto-garza How does this note look on your side?
> 
> ```lisp
> ---
> title: Actual frontmatter title
> ---
> 
> # Title
> 
> ***
> 
> Text a b c
> 
> ***
> ```
It looks like this:

![image](https://user-images.githubusercontent.com/8660999/62885656-06e71280-bcff-11e9-9a78-e389adc4d615.png)

**Comment by Flexo013** [comment]
Even with those exact settings I can't reproduce this bug. I have no idea where this could come from, so I'm just going to ask some random questions. Maybe that'll help in pinpointing the cause of this.

- Does this happen on all notes? (New, old, imported, exported)
- Do you have a different Win10 machine that you could test this behavior on?
- Could you try reinstalling Boostnote entirely? (Make sure to back up your notes first!)
- Has this bug occurred in previous versions?
- How long have you been using Boostnote?
- What is this extra middle section here? (Is this just the 80 character limit line?)
![image](https://user-images.githubusercontent.com/4507015/62965709-e5bd1980-be05-11e9-99fb-148059daa2b6.png)

Hopefully this will help in figuring the bug out.

**Comment by humberto-garza** [comment]
> en with those exact settings I can't reproduce this bug. I have no idea where this could come from, so I'm just going to ask some random questions. Maybe that'll help in pinpointing the cause

I was able to find a way to reproduce the issue!
This only happens when the zoom option is set to 90%; when it is set to 80%, 100%, 110%... the problem goes away... So I think it is a zoom related problem

**Comment by Flexo013** [comment]
@humberto-garza Awesome! Thanks for locating the source of the bug.

I can reproduce this issue now:
![image](https://user-images.githubusercontent.com/4507015/63032014-5111f480-beb5-11e9-91c4-82b362a48900.png)

**Comment by Flexo013** [comment]
Solved as of 0.15.0
